presto-hive/src/main/java/com/facebook/presto/hive/OrcFileWriterConfig.java

```diff
@@ -20,94 +20,95 @@
 @SuppressWarnings("unused")
 public class OrcFileWriterConfig
 {
-    private OrcWriterOptions options = new OrcWriterOptions();
+    private final OrcWriterOptions.Builder optionsBuilder = OrcWriterOptions.builder();
 
-    public OrcWriterOptions toOrcWriterOptions()
+    public OrcWriterOptions.Builder toOrcWriterOptionsBuilder()
     {
-        return options;
+        // Give separate copy to callers for isolation.
+        return optionsBuilder.copy();
```

Comment:
Oh, now I see why do you need the `copy` and the getters. Basically you are trying to avoid redeclaring all the same fields in the `OrcFileWriterConfig`. But honestly I don't think that's a big deal, and it feels like it could be a reasonable treadoff for a more intuitive builder interface. How strongly do you feel against simply copying the fields into this configuration? (it would also allow you to decouple `OrcWriterOptions` from the Presto specific configs)

Reply:
I have copied the fields to the config. This comes with a change in behavior, if OrcFileWriterConfig is mutated, previously OrcFileWriterConfig used to work with the old data before mutation. Now the mutation will be reflected. But looking at the code, it is not mutated once created, so this should be fine.